storage.go

```diff
@@ -560,6 +616,27 @@ func (s *Storage) stat(ctx context.Context, path string, opt pairStorageStat) (o
 		return nil, err
 	}
 
+	metadata := output.Metadata
+	for k, v := range metadata {
+		if k == "x-amz-meta-bs-symlink" {
```

Comment:
How about use `bs-link-target` instead of `bs-symlink`? There is no symlink in go-storage.

And we can extract it as a const value.

Reply:
Got it.